public/scripts/ebayAuction.js

```diff
@@ -0,0 +1,29 @@
+(function(module){
+  var auction = {};
+  auction.all=[];
+
+  auction.getAuction = function() {
+    for (var i = 0; i < artist.all.length; i++) {
+      // this regex replaces white space with '%20' which is needed for url to work
+      artist.all[i].name = artist.all[i].name.replace(/\s+/g, '%20');
+      var url = "http://svcs.ebay.com/services/search/FindingService/v1";
+      url += "?OPERATION-NAME=findItemsAdvanced";
+      url += "&categoryId=176985";
+      url += "&SERVICE-VERSION=1.0.0";
+      url += "&SECURITY-APPNAME=LauraSam-Code301-PRD-1e805b329-262cd85d";
```

Comment:
I think Laura and I already confirmed this, but make sure that this SECURITY-APPNAME param does not grant access to others to use your API key or access your own account information/functionality.